src/predict.rs

```diff
@@ -61,9 +61,11 @@ pub static RAV1E_INTER_COMPOUND_MODES: &[PredictionMode] = &[
   PredictionMode::NEW_NEWMV,
   PredictionMode::NEAREST_NEWMV,
   PredictionMode::NEW_NEARESTMV,
-  PredictionMode::NEAR_NEARMV,
+  PredictionMode::NEAR_NEAR0MV,
```

Comment:
Should this be updated to add the new modes?

Reply:
The new modes will be added in #2237, which extends the search. This PR is for preparing the accounting.